parm/config/config.ufs

```diff
@@ -0,0 +1,356 @@
+        export WRTIOBUF="48M"
+        ;;
+    "C3072")
+        export DELTIM=90
+        export layout_x=16
+        export layout_y=32
+        export layout_x_gfs=16
+        export layout_y_gfs=32
+        export nth_fv3=4
+        export nth_fv3_gfs=4
+        export cdmbgwd="4.0,0.05,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export WRITE_GROUP=4
+        export WRTTASK_PER_GROUP=$(echo "3*${npe_node_max}" |bc)  # TODO: this is pointless because below this is getting reset
+        export WRITE_GROUP_GFS=4
+        export WRTTASK_PER_GROUP_GFS=$(echo "3*${npe_node_max}" |bc)  # TODO: this is pointless because below this is getting reset
+        export WRTIOBUF="64M"
+        ;;
+    *)
+        echo "FATAL ERROR: Unsupported FV3 resolution = ${fv3_res}, ABORT!"
+        exit 1
+        ;;
+esac
+
+if [[ "${WRTTASK_PER_GROUP}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP=${npe_node_max} ; fi
+if [[ "${WRTTASK_PER_GROUP_GFS}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP_GFS=${npe_node_max} ; fi
+
+(( ntasks_fv3 = layout_x * layout_y * 6 ))
+(( ntasks_fv3_gfs = layout_x_gfs * layout_y_gfs * 6 ))
+export ntasks_fv3
+export ntasks_fv3_gfs
+
+(( ntasks_quilt = WRITE_GROUP * WRTTASK_PER_GROUP ))
+(( ntasks_quilt_gfs = WRITE_GROUP_GFS * WRTTASK_PER_GROUP_GFS ))
+export ntasks_quilt
+export ntasks_quilt_gfs
+
+# Determine whether to use parallel NetCDF based on resolution
+case ${fv3_res} in
+  "C48" | "C96" | "C192" | "C384")
+    export OUTPUT_FILETYPE_ATM="netcdf"
+    export OUTPUT_FILETYPE_SFC="netcdf"
+    ;;
+  "C768" | "C1152" | "C3072")
+    export OUTPUT_FILETYPE_ATM="netcdf_parallel"
+    export OUTPUT_FILETYPE_SFC="netcdf_parallel"
+    ;;
+esac
+
+# MOM6 specific settings
+if [[ "${skip_mom6}" == "false" ]]; then
+  export nth_mom6=1  # MOM6 is not thread-safe (allegedly)
+  case "${mom6_res}" in
+    "500")
+      ntasks_mom6=8
+      OCNTIM=3600
+      NX_GLB=72
+      NY_GLB=35
+      DT_DYNAM_MOM6='3600'
+      DT_THERM_MOM6='3600'
+      FRUNOFF=""
+      CHLCLIM="seawifs_1998-2006_smoothed_2X.nc"
+      MOM6_RESTART_SETTING='r'
+      MOM6_RIVER_RUNOFF='False'
+      ;;
+    "100")
+      ntasks_mom6=20
+      OCNTIM=3600
+      NX_GLB=360
+      NY_GLB=320
+      DT_DYNAM_MOM6='1800'
+      DT_THERM_MOM6='3600'
+      FRUNOFF=""
+      CHLCLIM="seawifs_1998-2006_smoothed_2X.nc"
+      MOM6_RESTART_SETTING='n'
+      MOM6_RIVER_RUNOFF='False'
+      ;;
+    "50")
+      ntasks_mom6=60
+      OCNTIM=3600
+      NX_GLB=720
+      NY_GLB=576
+      DT_DYNAM_MOM6='1800'
+      DT_THERM_MOM6='3600'
+      FRUNOFF="runoff.daitren.clim.${NX_GLB}x${NY_GLB}.v20180328.nc"
+      CHLCLIM="seawifs-clim-1997-2010.${NX_GLB}x${NY_GLB}.v20180328.nc"
+      MOM6_RESTART_SETTING='n'
+      MOM6_RIVER_RUNOFF='True'
+        ;;
+    "025")
+      ntasks_mom6=220
+      OCNTIM=1800
+      NX_GLB=1440
```

Comment:
Are these values also needed for ice? I know they were in the config.ice (probably in a few places tbh), but this section is "mom6 only" but I see NX_GLB/NY_GLB is needed for the ice model too in parsing_namelists_CICE

Reply:
From conversations w/ @DeniseWorthen, the domain size for ice should be the same as the ocean.  `NX_GLB` and `NY_GLB` need to be consistent when running CICE6 with MOM6.  
Previously, these were defined in `parsing_namelists_MOM.sh` for MOM6 and repeated in `config.ice` for ice.
This PR connects them in `config.ufs`

Reply:
@aerorahul my comment was related to the fact that these are in the "if [[ "${skip_mom6}" == "false" ]]; then" block.  They should be the same value --- I agree with your comment that we probably shouldn't be supporting running CICE without MOM6, but this is not clear by looking at this file alone so I wanted to point it out in case this should be a concern, but seems unlikely

Reply:
I added a check in the CICE block that will prevent running with CICE6 and without MOM6.